robyn/__init__.py

```diff
@@ -32,6 +33,21 @@
 
 __version__ = get_version()
 
+INFO_APP = 55  # Level number, higher than CRITICAL (50)
+logging.addLevelName(INFO_APP, "INFO_APP")  # Add level name
+
+
+def info_app(self, message, *args, **kwargs):
+    self._log(INFO_APP, message, args, kwargs)
+
+
+logging.Logger.info_app = info_app  # Add method to Logger class
+
+logging.INFO_APP = INFO_APP  # Optional - for convenience
+
+logger = logging.getLogger(__name__)
```

Comment:
Hey @carlosm27 👋 

Thank you for the PR 😄 Can you please explain what the `INFO_APP` parameter does?

Reply:
Hey @sansyrox it creates a new log level because with the `info` level shows also the logs from Actix too. But, I just notice in the Logger documentation that it doesn't advice creating new logs levels because it can create conflicts with other libraries. So, I will continue working on this and making updates to receive their review.

Reply:
Hey @carlosm27 any further updates to this?